source/jormungandr/jormungandr/scenarios/ridesharing/instant_system.py

```diff
@@ -51,6 +54,7 @@ def __init__(self, instance, service_url, api_key, network, rating_scale_min=Non
         self.rating_scale_max = rating_scale_max
         self.system_id = 'Instant System'
         self.timeout = timeout
+        self.feed_publisher = None if feed_publisher is None else Rs_FeedPublisher(**feed_publisher)
```

Comment:
This should be in conf, and for the STAR (Rennes) will probably have url : https://instant-system.com/disclaimers/disclaimer_33.html